hebiros_gazebo_plugin/plugin/hebiros_gazebo_plugin.cpp

```diff
@@ -69,11 +116,10 @@ void HebirosGazeboPlugin::UpdateGroup(std::shared_ptr<HebirosGazeboGroup> hebiro
       int i = hebiros_joint->feedback_index;
 
       joint->SetProvideFeedback(true);
-      double position = joint->GetAngle(0).Radian();
       double velocity = joint->GetVelocity(0);
-      physics::JointWrench wrench = joint->GetForceTorque(0);
-      auto trans = joint->GetChild()->GetInitialRelativePose().rot;
-      double effort = (-1 * (trans * wrench.body1Torque)).z;
+
+      double position = GazeboWrapper::position(joint);
```

Comment:
It's awesome that this is the end result. The actual code changed for compatibility between the two versions is super small!

Reply:
:D I know!